docs/building-modules/handler.md

```diff
@@ -21,8 +21,8 @@ Let us break it down:
 
 - The [`Msg`](./messages-and-queries.md#messages) is the actual object being processed. 
 - The [`Context`](../core/context.md) contains all the necessary information needed to process the `msg`, as well as a cache-wrapped copy of the latest state. If the `msg` is succesfully processed, the modified version of the temporary state contained in the `ctx` will be written to the main state.
-- The [`Result`] returned to `baseapp`, which contains (among other things) information on the execution of the `handler`, [`gas`](../basics/gas-fees.md) consumption and [`events`](../core/events.md).
-	+++ https://github.com/cosmos/cosmos-sdk/blob/7d7821b9af132b0f6131640195326aa02b6751db/types/result.go#L15-L40
+- The [`*Result`] returned to `baseapp`, which contains (among other things) information on the execution of the `handler` and [`events`](../core/events.md).
+	+++ https://github.com/cosmos/cosmos-sdk/blob/d55c1a26657a0af937fa2273b38dcfa1bb3cff9f/proto/cosmos/base/abci/v1beta1/abci.proto#L81-L95
 
 ## Implementation of a module `handler`s
```

Comment:
I'd also add the telemetry metrics somewhere on this file 👍

Reply:
we already have https://github.com/cosmos/cosmos-sdk/blob/master/docs/core/telemetry.md, what exactly would you like to add about metrics in `handler.md`?

Reply:
just mention how to create new metrics (one for each type). For example:

```golang
	defer func() {
		for _, a := range msg.Amount {
			telemetry.SetGaugeWithLabels(
				[]string{"tx", "msg", "send"},
				float32(a.Amount.Int64()),
				[]metrics.Label{telemetry.NewLabel("denom", a.Denom)},
			)
		}
	}()
```

Reply:
added, let me know if I missed something.